api/src/main/java/org/apache/iceberg/ManifestFile.java

```diff
@@ -177,6 +177,15 @@ default boolean hasDeletedFiles() {
   /** Returns the total number of rows in all files with status DELETED in the manifest file. */
   Long deletedRowsCount();
 
+  /**
+   * Returns true if there are any added or existing files
+   *
+   * @return whether this manifest contains entries with ADDED and/or EXISTING
+   */
+  default boolean hasLiveFiles() {
+    return this.hasAddedFiles() || this.hasExistingFiles();
```

Comment:
can probably drop `this` for consistency as it's not used for all methods in this class.